lib/God/Methods.js

```diff
@@ -242,7 +257,7 @@ module.exports = function(God) {
         if (mode.indexOf('cluster') === 0)
           treekill(parseInt(pid));
         else
-          process.kill(-(parseInt(pid)), 'SIGINT');
```

Comment:
I think I explained this in https://github.com/Unitech/pm2/issues/1036#issuecomment-91133953. Basically, `cluster` doesn't accept a `{detached: true}` option (like `fork`) and causes it no to be attached to it's own child processes.